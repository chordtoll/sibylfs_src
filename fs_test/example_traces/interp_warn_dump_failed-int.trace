@type script
dump /gone
dump /not_there
