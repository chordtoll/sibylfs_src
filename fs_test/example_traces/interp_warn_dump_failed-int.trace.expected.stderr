Error: error while dumping fs-state: Dump.Dump_error("ls_path")

Error:      2: dump "/gone"
   execution of dump failed
Error: error while dumping fs-state: Dump.Dump_error("ls_path")

Error:      3: dump "/not_there"
   execution of dump failed
Warning: trace caused problems
